### PR TITLE
Add recently submitted shootouts

### PR DESCRIPTION
**Pull request by benharsh**

### CODE REVIEW

File: test/studies/shootout/submitted/knucleotide.compopts [hunk 1]
```diff
@@ -0,0 +1 @@
+--no-warnings
```
Comment (ronawho) [inline test/studies/shootout/submitted/knucleotide.compopts]: Why is this needed?

File: test/studies/shootout/submitted/knucleotide.execopts [hunk 2]
```diff
@@ -0,0 +1 @@
+--n=0 < fasta-knucleotide.small
```
Comment (ronawho) [inline test/studies/shootout/submitted/knucleotide.execopts]: Do you know why these tests do < instead of using a .stdin file?

File: test/studies/shootout/submitted/knucleotide.compopts (hunk 1 above)
Comment (benharsh) [inline test/studies/shootout/submitted/knucleotide.compopts]: I think it's to avoid the warning about 'QuickSort' vs 'quickSort'

File: test/studies/shootout/submitted/knucleotide.execopts (hunk 2 above)
Comment (benharsh) [inline test/studies/shootout/submitted/knucleotide.execopts]: Nope, I had forgotten that was even an option.

**Comment by ronawho** [comment]
Looks good. It's a shame to need to copy all the .compopts, skipifs and suppressions, but I guess that's what we're stuck with for now
